subsys/bluetooth/audio/cap_common.c

```diff
@@ -0,0 +1,345 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/logging/log.h>
+
+#include "cap_internal.h"
+#include "csip_internal.h"
+
+#if defined(CONFIG_BT_CAP_INITIATOR) && defined(CONFIG_BT_CAP_COMMANDER)
+/* Use the highest debug level of the two */
+#if CONFIG_BT_CAP_INITIATOR_LOG_LEVEL > CONFIG_BT_CAP_COMMANDER_LOG_LEVEL
+LOG_MODULE_REGISTER(bt_cap_common, CONFIG_BT_CAP_INITIATOR_LOG_LEVEL);
+#else
+LOG_MODULE_REGISTER(bt_cap_common, CONFIG_BT_CAP_COMMANDER_LOG_LEVEL);
+#endif /* CONFIG_BT_CAP_INITIATOR_LOG_LEVEL > CONFIG_BT_CAP_COMMANDER_LOG_LEVEL */
+#elif defined(CONFIG_BT_CAP_INITIATOR)
+LOG_MODULE_REGISTER(bt_cap_common, CONFIG_BT_CAP_INITIATOR_LOG_LEVEL);
+#elif defined(CONFIG_BT_CAP_COMMANDER)
+LOG_MODULE_REGISTER(bt_cap_common, CONFIG_BT_CAP_COMMANDER_LOG_LEVEL);
+#endif /* CONFIG_BT_CAP_INITIATOR && CONFIG_BT_CAP_COMMANDER */
```

Comment:
This looks odd. Why don't you define a separate Kconfig logging option for this file?

Reply:
Added a CAP common log level